app/src/main/java/com/gmail/mxwild/newhabit/moviedetail/MovieDetailsViewModel.kt

```diff
@@ -0,0 +1,16 @@
+package com.gmail.mxwild.newhabit.moviedetail
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.gmail.mxwild.newhabit.model.data.Movie
+
+class MovieDetailsViewModel(private val movie: Movie) : ViewModel() {
+
+    private val _selectedMovie = MutableLiveData<Movie>()
+    val selectedMovie: LiveData<Movie> get() = _selectedMovie
+
+    fun setMovie() {
+        _selectedMovie.value = movie
```

Comment:
You can use movie in the mutable live data initialisation, so you don't need to keep it, and you don't need to have method setMovie.

Reply:
You right. I'm solved it